src/data_provider/src/sysInfoLinux.cpp

```diff
@@ -355,6 +364,118 @@ nlohmann::json SysInfo::getNetworks() const
     return networks;
 }
 
+std::pair<std::string, std::string> parseProcessInfo(char path[])
+{
+    // Open stat file
+    std::pair<std::string, std::string> processInfo;
+    std::ifstream file(path, std::ios::in | std::ios::binary);
+    std::string statContent;
+
+    // Get stat file content
+    std::getline(file, statContent);
+
+    // Parse PID and process name
+    int openParentesysPos = std::string(statContent).find("(");
+    int closeParentesysPos = std::string(statContent).find(")");
+    int spacePos = std::string(statContent).find(" ");
+
+    processInfo.first = std::string(statContent).substr(0, spacePos);
```

Comment:
For this section, have you considered using the string helpers available?

Something like `Utils::split()` for example to get all the tokens, and then `Utils::replaceAll()` to remove the `()`.

Reply:
Wouldn't that way add some overhead to the implementation? Now I'm just managing indexes without manipulating strings until the end.